client/me/security-2fa-key/style.scss

```diff
@@ -0,0 +1,43 @@
+.security-2fa-key__add-wait-for-key {
+	text-align: center;
+
+	.spinner {
+		margin-bottom: 1em;
+	}
+
+	p {
+		margin-bottom: 0.5em;
+	}
+}
+
+.security-2fa-key__add-button-container {
+	display: flex;
+	justify-content: flex-end;
+	margin-top: 2em;
+}
+
+.security-2fa-key__list {
+	list-style-type: none;
+	margin: 0;
+}
+
+.security-2fa-key__item {
+	display: flex;
+	align-items: center;
+
+	.security-2fa-key__item-information {
```

Comment:
No need to nest these classes (`.security-2fa-key__item-information`, `.security-2fa-key__item-subtitle` and `.security-2fa-key__delete-key`).

Reply:
Fixed in 811f6bd